config/routes.rb

```diff
@@ -2,7 +2,13 @@
 
 Rails.application.routes.draw do
   resources :kudos
-  devise_for :employees
+  devise_for :admin_users, path: 'admin_users'
+  devise_for :employees, path: 'employees'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'kudos#index'
+  get '/admin/', to: 'pages#dashboard'
+
+  namespace :admin_users do
```

Comment:
according to Acceptance Criteria namespace should be named admin

```suggestion
  namespace :admin do
```